cardano-testnet/src/Testnet/Cardano.hs

```diff
@@ -27,14 +27,18 @@ import           Control.Monad
 import           Control.Monad.IO.Class (MonadIO, liftIO)
 import           Control.Monad.Trans.Except
 import qualified Data.Aeson as J
+import qualified Data.Aeson.Key as Key
+import           Data.Aeson.KeyMap (KeyMap)
+import qualified Data.Aeson.KeyMap as Aeson
 import qualified Data.ByteString as BS
-import           Data.ByteString.Lazy (ByteString)
+import qualified Data.ByteString.Lazy as LB
```

Comment:
Checking over the repository.  There are examples of `LB` and `LBS` being used, but `LBS` is used more.  Perhaps we can standardise on that?

Reply:
Definitely, I'll switch to LBS here and follow up with a PR to change LB -> LBS